src/backend/distributed/commands/drop_distributed_table.c

```diff
@@ -38,6 +38,8 @@ master_drop_distributed_table_metadata(PG_FUNCTION_ARGS)
 	char *schemaName = text_to_cstring(schemaNameText);
 	char *tableName = text_to_cstring(tableNameText);
 
+	EnsureSchemaNode();
```

Comment:
I think that was my mistake where I put the comment. Please see [this](https://github.com/citusdata/citus/pull/1068#discussion_r94301491) comment